tests/Monolog/Formatter/JsonFormatterTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\ErrorControl\Tests\Monolog\Formatter;
+
+use DateTimeImmutable;
+use Exception;
+use Keboola\ErrorControl\Monolog\Formatter\JsonFormatter;
+use Monolog\Level;
+use Monolog\LogRecord;
+use PHPUnit\Framework\TestCase;
+
+class JsonFormatterTest extends TestCase
+{
+    public function testFormattingException(): void
+    {
+        $logWithException = new LogRecord(
+            new DateTimeImmutable(),
+            'test',
+            Level::Critical,
+            'Message',
+            [new Exception('Message', previous: new Exception('Previous'))]
+        );
+
+        $json = json_decode((new JsonFormatter())->format($logWithException), true);
+        assert(is_array($json));
+
+        $context = $json['context'][0];
+        self::assertEquals('Exception', $context['class']);
+        self::assertEquals('Message', $context['message']);
+        self::assertEquals(0, $context['code']);
+        self::assertMatchesRegularExpression('/^.*\/JsonFormatterTest\.php:\d+/', $context['file']);
+        self::assertNotEmpty($context['trace']);
```

Comment:
No ten duvod, proc jsem chtel porovnavat cely ten context je, aby bylo videt, ze ten `trace` je jeden velky string. Timhle ti projde i to pole, ktery dela originalni `JsonFormatter`. Tohle je asi jedina vec, kterou bych chtel, jeslti by si upravil. Aspon teda assert, ze je to neprazdny string. (abysme meli posichrovany, ze to nebude to pole)